invenio_cli/helpers/cli_config.py

```diff
@@ -61,6 +62,26 @@ def __init__(self, project_dir="./"):
             with open(self.private_config_path) as cfg_file:
                 self.private_config.read_file(cfg_file)
 
+    @property
+    def python_package_manager(self) -> PythonPackageManager:
+        """Get python packages manager."""
+        manager_name = self.config[CLIConfig.CLI_SECTION].get(
+            "python_package_manager", None
+        )
+        if manager_name == Pipenv.name:
+            return Pipenv()
+        elif manager_name == UV.name:
+            return UV()
+
+        if (self.project_path / "Pipfile").is_file():
+            return Pipenv()
+        elif (self.project_path / "pyproject.toml").is_file():
+            return UV()
```

Comment:
personally i've found this quite convenient as it's not _necessary_ to add the config item to `.invenio` (although if present, that one takes precedence)

Reply:
Agree, and works well since we're explicitly checking for the file on the top-level (and not e.g. when someone randomly runs `invenio-cli ...` accidentally inside the `site` folder).